assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -306,6 +306,13 @@ che.openshift.secure.routes=false
 che.openshift.jobs.image=centos:centos7
 che.openshift.jobs.memorylimit=250Mi
 
+# Run job to create workspace subpath directories in persistent volume before launching workspace.
+# Necessary in some versions of OpenShift/Kubernetes as workspace subpath volumemounts are created
+# with root permissions, and thus cannot be modified by workspaces running as user (presents as error
```

Comment:
You may want to link the issue https://github.com/kubernetes/kubernetes/issues/41638